app/models/miq_request_workflow.rb

```diff
@@ -1022,7 +1022,7 @@ def vm_or_template_to_hash_struct(ci)
   end
 
   def ems_folder_to_hash_struct(ci)
-    build_ci_hash_struct(ci, [:name, :is_datacenter?])
+    build_ci_hash_struct(ci, [:name, :type])
```

Comment:
@agrare Looks like you properly corrected the two `is_datacenter?` checks used by the request workflow.  I am good with this change.